upup/models/cloudup/resources/addons/networking.projectcalico.org/v2.0.yaml.template

```diff
@@ -77,7 +77,7 @@ spec:
         # container programs network policy and routes on each
         # host.
         - name: calico-node
-          image: quay.io/calico/node:v1.0.0
+          image: calico/node:v1.0.1
```

Comment:
You run a smoke test against k8s 1.5.2 and a master build?  Right now we are not running e2e testing on the cni providers.